devnet/docker-compose-substrate.yml

```diff
@@ -0,0 +1,57 @@
+services:
+  prometheus:
```

Comment:
why are we running prometheus in this docker-compose?

Reply:
I added Prometheus while testing the exported metric data from authority nodes, I decided to keep it in the same file to easily start a Prometheus server with authorities nodes, should I remove it?